cloudrun-malware-scanner/config.json

```diff
@@ -9,12 +9,5 @@
     "'ClamCvdMirrorBucket' is a GCS bucket used to mirror the clamav database definition files to prevent overloading the Clam servers",
     "and being rate limited/blacklisted. Its contents are maintained by the updateCvdMirror.sh script"
   ],
-  "buckets": [
-    {
-      "unscanned": "unscanned-bucket-name",
-      "clean": "clean-bucket-name",
-      "quarantined": "quarantined-bucket-name"
-    }
-  ],
-  "ClamCvdMirrorBucket": "cvd-mirror-bucket-name"
+  "ClamCvdMirrorBucket": "cvd-mirror-development-344423"
```

Comment:
why is this set to cvd-mirror-development-344423?

Reply:
The name can be anything. By default, it's cvd-mirror-$PROJECT_ID
This bucket is used to maintain a local mirror of the malware definitions database, which prevents rate limiting from being triggered by the ClamAV CDN.
I haven't setup this pipeline yet.

Reply:
i am saying we cant have a default here
it can just take the bucket from the input request
or on server boot up (maybe env var, or startup arg)

Reply:
Could you tell more about why we can't have a default here?
This bucket is created by me. It's just some bucket to hold the mirror of the db.

Reply:
Done. Put the config into environment variables

Reply:
i would still like to remove this default
so that if CVD_MIRROR_BUCKET is not set, this service does not boot

Reply:
Yes, it is like this now.
if CVD_MIRROR_BUCKET is not set, this service does not boot

Reply:
oh.. then i am confused what this is for then

Reply:
Server won't read from this config file. I can remove this for readability, it doesn't affect functionality
When docker start up, it will read the environment variable and overwrite this json value.
Other clamav config files will generate base on this value.

Reply:
oh.. then can we just remove this?